src/agent/src/rpc.rs

```diff
@@ -1682,7 +1682,10 @@ fn is_signal_handled(proc_status_file: &str, signum: u32) -> bool {
                 warn!(sl!(), "parse the SigCgt field failed");
                 return false;
             }
-            let sig_cgt_str = mask_vec[1];
+            let mut sig_cgt_str = mask_vec[1];
+            if !sig_cgt_str.chars().nth(0).unwrap().is_numeric() {
+                sig_cgt_str = &sig_cgt_str[1..sig_cgt_str.len()-1]
```

Comment:
@Champ-Goblem if you end up sticking with this approach could you please update the format? Our static check failed on `cargo fmt -- --check`
```
-                sig_cgt_str = &sig_cgt_str[1..sig_cgt_str.len()-1]
+                sig_cgt_str = &sig_cgt_str[1..sig_cgt_str.len() - 1]
```

Reply:
I ended up using .trim in the end but I ran this against the new commit and there should be no issues now with formatting